src/test/scala/tech/cryptonomic/conseil/metadata/MetadataServiceTest.scala

```diff
@@ -62,9 +71,22 @@ class MetadataServiceTest extends WordSpec with Matchers with ScalatestRouteTest
       result shouldBe Some(List(Network("mainnet", "overwritten name", "tezos", "mainnet")))
     }
 
+    "override description for a network" in {
+      // given
+      val overriddenConfiguration = Map("tezos" ->
+        PlatformConfiguration(None, Some(true), None, Map("mainnet" ->
+          NetworkConfiguration(Some("overwritten name"), Some(true), Some("description")))))
+
+      // when
+      val result = sut(overriddenConfiguration).getNetworks(PlatformPath("tezos"))
+
+      // then
+      result shouldBe Some(List(Network("mainnet", "overwritten name", "tezos", "mainnet", Some("description"))))
+    }
+
     "filter out a hidden network" in {
       // given
-      val overriddenConfiguration = Map("tezos" -> PlatformConfiguration(None, Some(true), Map("mainnet" -> NetworkConfiguration(None, Some(false)))))
+      val overriddenConfiguration = Map("tezos" -> PlatformConfiguration(None, Some(true), None, Map("mainnet" -> NetworkConfiguration(None, Some(false)))))
```

Comment:
would it be better to use named parameters with defaults to avoid writing all those `None`, `Some` which doesn't look so obvious to follow?